process/sync/baseSync.go

```diff
@@ -695,6 +700,14 @@ func (boot *baseBootstrap) rollBack(revertUsingForkNonce bool) error {
 			)
 		}
 
+		err = process.SetScheduledSCRs(prevHeaderHash, boot.store, boot.marshalizer, boot.scheduledTxsExecutionHandler)
+		if err != nil {
+			log.Debug("set scheduled scrs",
```

Comment:
shouldn't this be a critical error ? or at least log.warn / log.error

Reply:
Actually there will be probably less than 1% of blocks with scheduled txs and scrs in the normal cases, so these headers will have nothing under their hashes in the ScheduledSCRsStorage and this will cause that this method to return in > 99% cases error like "...key not found...", which is not a real error. 

Anyway if something goes wrong for the node when it loading any kind of stuff from storage like: meta blocks, pending mini blocks, scheduled scrs, it will conduct to a stuck situation for it, as something will surely not match after the next block processing and so, it will try to roll back more blocks.